isis/src/qisis/objs/ControlNetTool/ControlNetTool.h

```diff
@@ -58,6 +58,8 @@ namespace Isis {
    *                           loaded.
    *   @history 2018-04-13 Tracie Sucharski - In mouseButtonRelease method return if a control net
    *                           has not been set.
+   *   @history 2018-09-21 Tracie Sucharski - Draw Fixed and Constrained points on ground cubes.
+   *                           Refereces #5504.
```

Comment:
spelling (I know it's picky, but since you're here)

Reply:
Corrected spelling of Reference.